examples/decoding/plot_receptive_field.py

```diff
@@ -0,0 +1,135 @@
+"""
+=========================================
+Receptive Field Estimation and Prediction
+=========================================
+
+This example reproduces figures from Lalor et al's mTRF toolbox in
+matlab [1]_. We will show how the :class:`mne.decoding.ReceptiveField` class
+can perform a similar function along with :mod:`sklearn`. We will fit a
+linear encoding model using the continuously-varying speech envelope to
+predict activity of a 128 channel EEG system.
+
+References
+----------
+.. [1] Crosse, M. J., Di Liberto, G. M., Bednar, A. & Lalor, E. C. (2016).
+       The Multivariate Temporal Response Function (mTRF) Toolbox:
+       A MATLAB Toolbox for Relating Neural Signals to Continuous Stimuli.
+       Frontiers in Human Neuroscience 10, 604. doi:10.3389/fnhum.2016.00604
+"""
+# Authors: Chris Holdgraf <choldgraf@gmail.com>
+#
+# License: BSD (3-clause)
+
+
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.io import loadmat
+from scipy.stats import pearsonr
+
+import mne
+from mne.decoding import ReceptiveField
+from sklearn.linear_model import Ridge
+from sklearn.model_selection import KFold
+from sklearn.preprocessing import scale
+
+
+###############################################################################
+# Load the data from the publication
+path = mne.datasets.mtrf.data_path()
+n_decim = 2  # To speed up computation
```

Comment:
Something more like:

We can speed up computation by downsampling the data since we know that our features are going to be primarily low-frequency in nature